core/src/main/java/io/cucumber/core/options/CommandlineOptions.java

```diff
@@ -0,0 +1,53 @@
+package io.cucumber.core.options;
+
+public class CommandlineOptions {
```

Comment:
- This class should be final. 
- Moved to `io.cucumber.core.cli`
- Annotated with `@Api` (same as the Main class)
- Add a private constructor to prevent instantiation

Reply:
And don't forget to add a line to the `CHANGELOG`

Reply:
@mpkorstanje good point on private constructor 👍 I've updated the PR as your suggestions, please help me review them. Thanks!